bin/ch-fromhost

```diff
@@ -53,17 +63,19 @@ Which files to inject (one or more required; can be repeated):
   -c, --cmd CMD    listed in the stdout of CMD
   -f, --file FILE  listed in file FILE
   -p, --path PATH  inject the file at PATH
-  --cray-mpi       Cray-enable MPICH/OpenMPI installed within the image
-  --nvidia         recommended by nVidia (via "nvidia-container-cli list")
+  -o, --ofi  PATH  add host compiled OFI DSO provider(s) to image libfabric
+  --host-ofi       ofi providers at path CH_FROMHOST_OFI
+  --nvidia         recommended by nVidia (via \"nvidia-container-cli list\")
 
 Destination within image:
 
-  -d, --dest DST   place following files in IMGDIR/DST, overriding inference
+  -d, --dest DST   place preceding files in IMGDIR/DST, overriding inference
```

Comment:
This seems confusing UI?

Reply:
Ah, yes, I read `place [the] following`, giving me the impressions the next options were impacted.